### PR TITLE
modify

### PR DESCRIPTION
**Pull request by esraa33-95**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -186,20 +186,20 @@
 
 
 
-//task 3
+//task 3.1
 Route::post('users',[Xcontroller::class,'my_data']);
```
Comment (Nora-Lootfy) [inline routes/web.php:190]: Hello Esraa,, 

the functionality of your code is correct, but i have a little note here, you have violated some standard when i download your project in my computer look at this warning message

```shell
Class App\Http\Controllers\xcontroller located in ./app/Http/Controllers/Xcontroller.php does not comply with psr-4 
autoloading standard (rule: App\ => ./app). Skipping.
```

This means you didn't follow PHP standard recommendations

Your class definition
```php
class xcontroller extends Controller
```
correct
```php
class XController extends Controller
```

File: routes/web.php [hunk 2]
```diff
@@ -186,20 +186,20 @@
...
 });
 
 
-// Route::post('get_data',function(Request $request){
-//     $name      =$request->input('name');
-//     $email     =$request->input('email');
-//     $message   =$request->input('msg');
-//     $subject   =$request->input('subject');
+Route::post('get_data',function(Request $request){
+    $name      =$request->input('name');
```
Comment (Nora-Lootfy) [inline routes/web.php]: here your code is correct also but there is easier syntax

for example
```php
$email     =$request->input('email');
```

more clear syntax
```php
$email     =$request->email;
```

or 
```php
$email     =$request['email'];
```

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
Good job ya Esraa, just see my notes

**Comment by esraa33-95** [comment]
ok ,updated now.

**Pull request merged**